bedrock/mozorg/templates/mozorg/home/includes/ctd-promo-de.html

```diff
@@ -69,7 +69,7 @@ <h2>Und was machen wir jetzt?</h2>
       Wir könnten dich nach deinem Lieblingsdino fragen, ein kleines Gedicht für dich schreiben, oder dir unseren Mobile Browser zeigen.
     </p>
 
-    <a href="{{ url('firefox.browsers.mobile.index') }}?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=ctd-de" class="mzp-c-button mzp-t-product" data-cta-text="Ich schau mir Firefox mal an" data-cta-type="button" data-cta-position="banner">Zeigt mir den Mobile Browser</a>
+    <a href="{{ url('firefox') }}?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=ctd-de" class="mzp-c-button mzp-t-product" data-cta-text="Ich schau mir Firefox mal an" data-cta-type="button" data-cta-position="banner">Zeigt mir den Mobile Browser</a>
```

Comment:
We should not be including UTM parameters on same site links. If we do this then clicking the link creates a new session in GA, which will artificially inflate page views.

We already have data attributes in place here, so the UTMs should not be needed.